lib/Button.js

```diff
@@ -74,82 +75,22 @@ export class Button extends Entity {
         this.accentColour = this.#pen.gui.accentColour;
     }
 
-    /**
-     * Returns true if the button is currently being pressed.
-     * @returns {boolean}
-     * @readonly
-     */
-    get down() {
-        if (this.exists === false) {
-            return false;
-        }
-        const buttonWasVisible =
-            this.#lastFrameDrawn === this.#pen.frameCount - 1;
-        // console.log("buttonWasVisible",buttonWasVisible);
-        if (buttonWasVisible) {
-            return false;
-        }
-        if (this.isHovered()) {
-            this.#down = this.#pen.mouse.leftDown;
-        } else {
-            this.#down = false;
-        }
-
-        return this.#down;
-    }
-
-    /**
-     * @param {boolean} value
-     * @readonly
-     * @throws {Error} If you try to set this property.
-     */
-    set down(value) {
```

Comment:
Can we keep the sets with the helpful feedback on these readonly properties? It'll be easier for a student to parse.

Reply:
Didn't realise I removed that, added it back in.